runtime/tests.py

```diff
@@ -0,0 +1,16 @@
+import contextlib
+from os import remove
+
+from django.test import SimpleTestCase, TestCase
+from rest_framework import status
+from rest_framework.test import APITestCase, APIClient
+
+from .replayer import Replayer
+from logs.models import Log
+
+class DemoTest(TestCase):
+    
+    """def test_demo_executions(self):
```

Comment:
Why do you have `"""`  at the start?

Reply:
Because at the moment the replayer part is not working, but if I have to put the prediction part in the core folder I can put the runtime folder in gitignore so I can wait to push it here

Reply:
I pushed it anyways because I wanted you to have last changes I made in core folder

Reply:
ok